core/src/test/java/org/jboss/jca/core/connectionmanager/pool/OnePoolNoTxDecrementCapacityMinPoolSizePolicyTestCase.java

```diff
@@ -23,6 +23,8 @@
 
 import org.jboss.jca.core.api.connectionmanager.pool.PoolStatistics;
 import org.jboss.jca.core.connectionmanager.pool.mcp.ManagedConnectionPool;
+import org.jboss.jca.core.connectionmanager.rar.SimpleConnection;
+import org.jboss.jca.core.connectionmanager.rar.SimpleConnectionImpl;
```

Comment:
No SimpleConnectionImpl - required methods needs to be exposed in SimpleConnection